.eslintrc

```diff
@@ -78,6 +78,6 @@
     "comma-spacing": ["error", { "before": false, "after": true }],
     "eol-last": "error",
 
-    "prettier/prettier": "error"
+    // "prettier/prettier": "error"
```

Comment:
temp will remove. used to VSD code's format-on-save but can't seem to get it to play nice with the linter files.

Reply:
I landed on using the Webstorm IDE with Solidity plugins, perhaps worth considering? 

(Although not perfect, refactoring large scale Solidity projects is substantially worse in every other Editor/IDE I tried)

Reply:
will try out webstorm IDE. Im a Github Copilot fanboy tho, and that only works with VSCode i think.

Reply:
https://plugins.jetbrains.com/plugin/17718-github-copilot